src/common/lib/DevTools.browser.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import {createDevTools} from 'redux-devtools';
```

Comment:
Yeah, imports should be sorted. There should be easy `Sort Lines (Case Sensitive)` command in your editor.